docs/source/guides/hot-reload.md

```diff
@@ -0,0 +1,65 @@
+# How to hot reload Iroha in a Docker container
+
+Here is the overall procedure for hot reloading Iroha in a Docker container:
+
+1. Build Iroha on your host OS:
+
+    ```bash
+    cargo build --release
+    ```
+
+    Check the [troubleshooting](#troubleshooting-static-linking) section if the build fails.
+
+2. Enter docker container:
+
+    ```bash
+    docker exec -it iroha-iroha0-1 bash
+    ```
+
+3. Copy Iroha to the current directory:
+
+    ```bash
+    cp root/soramitsu/iroha/target/x86_64-unknown-linux-musl/release/iroha .
+    ```
+
+4. (Optional) Make any modifications you need:
+
+    - [Recommit genesis block](#recommit-genesis-block-fresh-start)
+    - [Use custom configuration files](#use-custom-configuration-files)
+    - [Use custom environment variables](#use-custom-environment-variables)
+
+5. Exit docker container and restart it using `docker restart`.
+
+    **Note:** If you use the combination of `container down` and `container up`, any modifications you made on the previous step will be lost. Use `docker restart` to preserve changes.
+
+If you skip the optional step (step 4), the state of the blockchain after hot reload will be the same as it was before the Docker container was restarted.
+
+## Recommit genesis block (fresh start)
+
+To recommit a custom the genesis block, remove the previously stored information about the blocks before restarting the container:
+
+```bash
+rm blocks/*
+```
+
+The new genesis block will be automatically recommited upon container restart.
+
+## Use custom configuration files 
+
+To use custom configuration files, such as `config.json` or `genesis.json`, copy them to to `/` before restarting the Docker container.
```

Comment:
This is no longer true, since we removed the default configuration, Iroha will not look in its root, but instead in the `config/` subvolume. I made this comment on a different occasion.